### DISCUSSION TITLE
Danfoss Ally eTRV0103 is unsupported.

### DISCUSSION THREAD

**Discussion by lukaszWisniew** [Koenkk/zigbee2mqtt#20339] [Devices]
This topic has probably been discussed before. If so, please provide a link and close the thread.

However, if there hasn't been such a topic yet, I'll continue.

I already have one Danfoss Ally, and it is recognized as Zigbee Model eTRV0100.
I purchased another Danfoss Ally, but it is recognized as Zigbee Model eTRV0103, and Home Assistant indicates that it is unsupported.

What should I do next?

Below are the logs from connecting the device.

```
MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x048727fffe44f6a3","ieee_address":"0x048727fffe44f6a3"},"type":"device_joined"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x048727fffe44f6a3"},"type":"device_connected"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x048727fffe44f6a3","ieee_address":"0x048727fffe44f6a3"},"type":"device_announce"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x048727fffe44f6a3"},"type":"device_announced"}'
Received message from unsupported device with Zigbee model 'eTRV0103' and manufacturer name 'Danfoss'
```

**Comment by Koenkk** [comment]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Comment by lukaszWisniew** [comment]
It's working. Great job.
